excel/src/test/java/org/apache/metamodel/excel/ExcelDataContextTest.java

```diff
@@ -928,14 +970,15 @@ public void testInsertingValueOfValidColumnType() {
         assertTrue(dataSet.next());
     }
 
-    @Test(expected = MetaModelException.class)
+    @Test
```

Comment:
Why does this no longer throw the exception? Do we want it not to?

Reply:
It never threw an exception. Because we were using JUnit3 the expected exception part of the annotation was ignored.